app/views/users/_organizations_area.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if @user.organizations.present? %>
+  <div id="sidebar-organizations" class="widget">
+    <div class="widget-suggested-follows-container">
+      <header><h4>organizations</h4></header>
+      <div class="widget-body">
+        <% @user.organizations.find_each do |organization| %>
+          <div class="widget-user-pic">
+            <a href="/<%= organization.slug %>">
+              <img src="<%= ProfileImage.new(organization).get(90) %>" alt="<%= organization.name %> profile image">
```

Comment:
Every time I see this `ProfileImage` method, I think `width` should have been a keyword arg. In case you're ever looking for another PR... 😉

Reply:
Is this partial cached? We have to make external requests to get all of these images so it might be worth making sure they are cached with the new `organization_info_updated_at` timestamp for the user to prevent having to grab them every single time.

Reply:
@mstruve Yes, it is cached in the main view at `app/views/users/show.html.erb:196`.

Reply:
> @citizen428 Do I need to create an issue first before working on it?

That would be great, thank you! 🎉  

It's just a minor refactor, but I really think `get(width: 90)` is so much clearer than just `.get(90)` that it's worth it. Feel free to reference this comment thread from the issue for context.